src/org/usfirst/frc/team1922/robot/commands/TurnTo_Command.java

```diff
@@ -25,20 +27,40 @@ public TurnTo_Command(int direction) {
     @Override
     protected void initialize() {
         start = Robot.m_driveTrain.getAngle();
-        Robot.m_driveTrain.drive(direction*(.2), -direction*(.2));
+        if( direction >= 0 ){
+        	Robot.m_driveTrain.drive(.4,-.4);
+        }else{ 
+        	isLeft = true;
+        	Robot.m_driveTrain.drive(-.35,.35);
+        	
+        }
+        SmartDashboard.putString("Turn Direction", "" + direction);
+        SmartDashboard.putString("Start Start", "" + start);
+        
         SmartDashboard.putString("Turn_Command", "intialized");
     }
     
     @Override
     protected void execute() {
         //if(direction != -1);  
-            if(Robot.m_driveTrain.getAngle() <= start + angle) {
+    	  if(isLeft == false){
```

Comment:
2 things about this line. 
1. You don't need to compare a boolean value to a boolean constant. This can just be `if(!isLeft)`.
2. It's generally more readable when you if conditions are positive rather than negative. So it should be `if(isLeft)` and then just flip the blocks so that the else block handles turning right.